packages/nextjs-kit/tailwind.config.js

```diff
@@ -1,6 +1,12 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
 	content: ['./src/components/**/*.tsx'],
+	safelist: [
+		{
+			pattern: /grid-cols-([0-9])/,
+			variants: ['lg'],
+		},
+	],
```

Comment:
I'm skeptical that this is the best solution to the cols problem, the docs say this is a last resort kind of thing. Is there truly no better way?

Reply:
I wrestled with this for awhile and have yet to find a solution I think is better. I tried the helper function route outlined [here](https://stackoverflow.com/a/72268889) but that did not fix the problem. The only other way I found to get the dynamic columns working was by side stepping tailwind and adding an inline style like so
```
style={{
	gridTemplateColumns: `repeat(${cols}, minmax(0, 1fr))`,
}}
```
This solution didn't feel any better right and makes it more difficult to add a media query for breakpoint definitions.

Let me know what you think 👍